docs/website/meta/versions.md

```diff
@@ -14,28 +14,35 @@ The current stable version of Brooklyn is {{ site.brooklyn-stable-version }}:
 This documentation was generated {{ site.time | date_to_string }}.
 
 
-### Version History
+## Version History
 
-Apache Brooklyn has made the following releases (links are to the user guides):
+Apache Brooklyn has made the following releases:
 
-* **[0.7.0-incubating](/v/0.7.0-incubating/)**: New policies, more clouds, improved Windows support and many other improvements. Apache-endorsed binary release! (July 2015)
+* **0.7.0-incubating**: New policies, more clouds, improved Windows support and many other improvements. Apache-endorsed binary release! (July 2015)
   * [User guide](/v/0.7.0-incubating/), download via mirrors
     &laquo;<a href="https://www.apache.org/dyn/closer.cgi/incubator/brooklyn/apache-brooklyn-0.7.0-incubating/apache-brooklyn-0.7.0-incubating-bin.tar.gz">bin.tar.gz</a>&raquo;
     &laquo;<a href="https://www.apache.org/dyn/closer.cgi/incubator/brooklyn/apache-brooklyn-0.7.0-incubating/apache-brooklyn-0.7.0-incubating-bin.zip">bin.zip</a>&raquo;
     &laquo;<a href="https://www.apache.org/dyn/closer.cgi/incubator/brooklyn/apache-brooklyn-0.7.0-incubating/apache-brooklyn-0.7.0-incubating-src.tar.gz">src.tar.gz</a>&raquo;
     &laquo;<a href="https://www.apache.org/dyn/closer.cgi/incubator/brooklyn/apache-brooklyn-0.7.0-incubating/apache-brooklyn-0.7.0-incubating-src.zip">src.zip</a>&raquo;
 
-* **[0.7.0-M2-incubating](/v/0.7.0-M2-incubating/)**: YAML, persistence, Chef, Windows, Docker. The first Apache release! (December 2014)
+* **0.7.0-M2-incubating**: YAML, persistence, Chef, Windows, Docker. The first Apache release! (December 2014)
   * [User guide](/v/0.7.0-M2-incubating/), download via mirrors
     &laquo;<a href="https://www.apache.org/dyn/closer.cgi/incubator/brooklyn/0.7.0-M2-incubating/apache-brooklyn-0.7.0-M2-incubating.tar.gz">src.tar.gz</a>&raquo;
 
-* **[0.7.0-SNAPSHOT](/v/0.7.0-SNAPSHOT/)**: bleeding-edge (not voted on or endorsed by Apache!)
```

Comment:
I'd prefer us to have a link for the bleeding edge from here. It's not just for "developers"; it's for anyone who is willing to take the informed risk of trying out bleeding edge unofficial builds. One does not need to build the code to use the snapshot build - one can get the binary from the apache maven snapshots repo.

Also, this is for docs - it's nice to be able to point people at the upcoming docs for the next release, to tell them what is coming (e.g. as a pointer on the mailing list).